File: Projeto-TCC/ACS-Cadastro-Mobile/app/src/main/java/tcc/acs_cadastro_mobile/subModels/Responsible.java
package tcc.acs_cadastro_mobile.subModels;

import org.json.JSONException;
import org.json.JSONObject;

import java.io.Serializable;

import io.realm.RealmObject;
import tcc.acs_cadastro_mobile.Constants;

import static tcc.acs_cadastro_mobile.Constants.Citizen.BIRTH_DATE;
import static tcc.acs_cadastro_mobile.Constants.Citizen.NUM_SUS;
import static tcc.acs_cadastro_mobile.Constants.Citizen.RESPONSIBLE;

public class Responsible extends RealmObject  implements Serializable {
    private boolean isResponsible;
    private long numSus;
    private String birthDate;

    public Responsible() {
        this(false, 0, "");
    }

    public Responsible(boolean isResponsible, long numSus, String birthDate) {
        this.isResponsible = isResponsible;
        this.numSus = numSus;
        this.birthDate = birthDate;
    }

    public boolean isResponsible() {
        return isResponsible;
    }

    public void setResponsible(boolean responsible) {
        isResponsible = responsible;
    }

    public long getNumSus() {
        return numSus;
    }

    public void setNumSus(long numSus) {
        this.numSus = numSus;
    }

    public String getBirthDate() {
        return birthDate;
    }

    public void setBirthDate(String birthDate) {
        this.birthDate = birthDate;
    }

    public JSONObject asJson() throws JSONException {
        JSONObject json = new JSONObject();
        json.put(RESPONSIBLE.name(), isResponsible);
        json.put(NUM_SUS.name(), numSus);
        json.put(BIRTH_DATE.name(), birthDate);
        return json;
    }
}

File: Projeto-TCC/ACS-Cadastro-Web/subModels/Responsible.php
<?php

class Responsible
{
    const RESPONSIBLE = "RESPONSIBLE";
    const NUM_SUS = "NUM_SUS";
    const BIRTH_DATE = "BIRTH_DATE";

    private $responsible, $numSus, $birthDate;

    function __construct($responsible, $numSus, $birthDate)
    {
        $this->responsible = $responsible;
        $this->numSus = $numSus;
        $this->birthDate = $birthDate;
    }

    public function getValuesToDB()
    {
        return array(":" . self::RESPONSIBLE => $this->responsible,
            ":" . self::NUM_SUS => $this->numSus,
            ":" . self::BIRTH_DATE => $this->birthDate);
    }

    public static function getFromArray(array $array)
    {
        return new Responsible($array[self::RESPONSIBLE], $array[self::NUM_SUS], $array[self::BIRTH_DATE]);
    }
}